server/src/collection.rs

```diff
@@ -257,4 +306,23 @@ impl Collection {
 
         Ok(())
     }
+
+    pub async fn delete_item(&self, path: &OwnedObjectPath) -> Result<(), Error> {
+        let Some(item) = self.item_from_path(path).await else {
+            tracing::debug!("Item: {} does not exist.", path);
+            return Err(Error::NoSuchObject);
+        };
+
+        let attributes = item.attributes().await;
+        self.keyring.delete(&attributes).await?; // TODO: we need a new method?
```

Comment:
For what?

Reply:
Sorry for the delay.

So, the issue is `delete` remove all the items that matches the attributes. All the items created using Seahorse have the same attributes: `"xdg:schema" "org.gnome.keyring.Note"`. So, using this method removes everything :)

Reply:
The same happens with Key Rack.

Reply:
Please open an issue then. We have to fix this asap.